### PR TITLE
Add hint how to disable ESLint

### LINKED ISSUE #17986: Add note about how to disable eslint in gatsby
I have deleted all my prettier and .eslint files, but when I fun gatsby develop I still get erros about linting. 

I don't want to use any linting at all. How do I disable it??

**Edit**:

Maybe we should add a small note at the bottom of https://www.gatsbyjs.org/docs/eslint/ explaining how to disable it?

### PR DESCRIPTION
**Pull request by ThomasJanUta**
## Description

Add a short paragraph at the end of the ESLint docs. It explains how to disable the implicit ESLinting behavior of Gatsby and how to disable it by adding an empty ESLint file as laid out by Michal Piechowiak (pieh) in the related issue.

## Related Issues

Fixes #17986

### CODE REVIEW

File: docs/docs/eslint.md [hunk 1]
```diff
@@ -38,4 +38,8 @@ module.exports = {
 }
 ```
 
-If you want to disable ESLint completely, just create an empty `.eslintrc` file.
+### Disabling ESLint
+
+Create an empty .eslintrc.js file to disable linting.
```
[the remark this answers is not in the document]
Comment (laurieontech) [reply docs/docs/eslint.md]: I'd keep this wording the same. It directs the user how to accomplish the task with the casual and friendly "you".

[the remark this answers is not in the document]
Comment (ThomasJanUta) [reply docs/docs/eslint.md]: Added the change

**Comment by laurieontech** [comment]
Hey @ThomasJanUta not sure what happened here. I'm seeing that the most recent edits remove your explanatory paragraph and don't account for the note above. Can you take a look at your most recent commit and make sure that's what you meant to do? Thanks for your help here!

**Comment by ThomasJanUta** [comment]
@laurieontech I added the explanatory paragraph back. Hope it's fine now! 🎉

**Comment by laurieontech** [comment]
Perfect! Thanks for the PR. Will get this merged in.

**Pull request merged**